File: src/h_classes/classes.java
package h_classes;

public class classes {
    public static void main(String[] args) {
        var user = new JavaUserRecord("Alice", "+49-012345678910");
        System.out.println(user.name());
        System.out.println(user.phoneNumber());

        var user2 = new JavaUserRegularClass("Bob", "+49-012345678910");
        user2.printUser();
    }
}

File: src/h_classes/classes.kt
package h_classes

fun main(args: Array<String>) {
    val user = KotlinUserDataClass(name = "Alice", phoneNumber = "+49-012345678910")
    println(user.name)
    println(user.phoneNumber)

    val user2 = KotlinUserRegularClass(name = "Bob", phoneNumber = "+49-012345678911")
    user2.printUser()
}
